src/main/kotlin/org/wfanet/measurement/common/crypto/SecurityProvider.kt

```diff
@@ -40,15 +39,17 @@ private val certFactory = CertificateFactory.getInstance(CERTIFICATE_TYPE, jcePr
 fun readCertificate(pemFile: File): X509Certificate {
   return pemFile.inputStream().use { fileInputStream ->
     certFactory.generateCertificate(fileInputStream)
-  } as X509Certificate
+  } as
+    X509Certificate
```

Comment:
This seems like bad formatting -- is the `as X509Certificate` actually necessary?

Reply:
The `as X509Certificate` is necessary, as the Java security API has a single `CertificateFactory` interface despite different instances returning more specific certificate types. We want to restrict our API to `X509Certificate` as opposed to other certificate types.